instance/instance_external_test.go

```diff
@@ -34,7 +34,7 @@ func TestGetInstancesReturnsOK(t *testing.T) {
 		w := httptest.NewRecorder()
 
 		mockedDataStore := &storetest.StorerMock{
-			GetInstancesFunc: func(string) (*models.InstanceResults, error) {
+			GetInstancesFunc: func([]string) (*models.InstanceResults, error) {
```

Comment:
You should probably add new test cases to cover single and multiple values being requested.

Reply:
Done 👍
https://github.com/ONSdigital/dp-dataset-api/pull/43/commits/1c1d4dcf537ad3cbbd49c34456d437b3171e2d80